share/nisar/defaults/gcov.yaml

```diff
@@ -570,14 +570,27 @@ runconfig:
                 # Default: True
                 qa_reports: true
 
+            software_config:
+
+                # True to use memory map(s) to cache select Dataset(s).
+                # False to always read data directly from the input file.
+                # Generally, enabling caching should reduce runtime.
+                # Default: True
+                use_cache: true
+
+                # True to delete the nested QA scratch directory in
+                # `scratch_path` and its contents when QA SAS is finished.
+                # Default: True
+                delete_qa_scratch_files: true
+
             validation:
 
-                # True to raise an exception if one or more metadata cubes contains
-                # all non-finite (e.g. Nan, +/- Inf) values, or if one or more
-                # z-dimension height layers in a 3D cube has all non-finite values.
+                # True to raise an exception if one or more metadata LUTs
+                # contain all non-finite (e.g. Nan, +/- Inf) values, or if one or more
+                # z-dimension height layers in a 3D LUT has all non-finite values.
                 # False to quiet the exception (although it will still be logged).
                 # Default: True
-                metadata_cubes_fail_if_all_nan: true
+                metadata_luts_fail_if_all_nan: true
```

Comment:
Does this flag apply to both LUTs and metadata cubes? I understand metadata cubes are a type of 3D LUT, but that’s not the terminology we’ve used in our documentation, code, or this YAML file. So, it may be worth clarifying that this also applies to metadata cubes (or radar grid cubes, in the case of L2 products).

Reply:
QA adopted the convention of "LUT" as the generic for all metadata look-up tables, which includes metadata cubes. PR comment: https://github.com/isce-framework/nisarqa/pull/9#discussion_r1932748405